### PR TITLE
testing/gitlab-runner: upgrade to 12.1.0

### PR DESCRIPTION
**Pull request by Cogitri**
* Add -openrc subpkg
* Add gitlab-runner user
* Cleanup

### CODE REVIEW

File: testing/gitlab-runner/gitlab-runner.initd [hunk 1]
```diff
@@ -0,0 +1,15 @@
+#!/sbin/openrc-run
+
+supervisor=supervise-daemon
+
+description="GitLab CI Runner"
+name="Gitlab Runner"
+
+command="/usr/bin/gitlab-runner"
+command_args="run ${GITLAB_RUNNER_OPTS}"
+command_background="yes"
```
Comment (maxice8) [inline testing/gitlab-runner/gitlab-runner.initd]: don't use command_background when using supervise daemon

Comment (Cogitri) [reply testing/gitlab-runner/gitlab-runner.initd]: Fixed, thanks

File: testing/gitlab-runner/gitlab-runner.initd [hunk 2]
```diff
@@ -0,0 +1,14 @@
+#!/sbin/openrc-run
+
+supervisor=supervise-daemon
+
+description="GitLab CI Runner"
+name="Gitlab Runner"
+
+command="/usr/bin/gitlab-runner"
+command_args="run ${GITLAB_RUNNER_OPTS}"
+command_user="${GITLAB_RUNNER_USER:-gitlab-runner}:${GITLAB_RUNNER_GROUP:-gitlab-runner}"
```
Comment (maxice8) [inline testing/gitlab-runner/gitlab-runner.initd]: Why have a default set here if you provide a default conf.d file ?

**Comment by Cogitri** [comment]
wfm now

**Pull request closed without merging**